roles/openstack_infra/tasks/cleanup.yml

```diff
@@ -0,0 +1,49 @@
+- name: delete master node
+  os_server:
+    name: master
+    state: absent
+    wait: yes
+- name: delete worker0
+  os_server:
+    name: worker0
+    state: absent
+    wait: yes
+- name: delete worker1
+  os_server:
+    name: worker1
+    state: absent
+    wait: yes
+- name: delete worker2
+  os_server:
+    name: worker2
+    state: absent
+    wait: yes
+- name: delete custom security group
+  os_security_group:
+    state: absent
+    name: manila-kube-security-group
+- name: delete keypair
```

Comment:
don't delete the keypair - it's not created by us here

Reply:
It is created by us https://github.com/tombarron/manila-kube/blob/master/roles/openstack_infra/tasks/main.yml#L34

Reply:
Ok